urth_dash_js/notebook/dashboard-view/sub-menu.html

```diff
@@ -3,10 +3,14 @@
     <a href="#">Dashboard</a>
     <ul class="dropdown-menu">
         <li id="urth-dashboard-show-all" class="disabled" title="Return all hidden cells to the Dashboard">
-            <a href="#">Show All Cells</a>
+            <a href="#">Show All Cells (Packed)</a>
+        </li>
+        <li id="urth-dashboard-show-all-stacked" class="disabled" title="Return all hidden cells to the Dashboard">
```

Comment:
You should update the `title` attribute to differentiate between the two layout strategies. These show when you hover over menu items.